modules/post/windows/gather/enum_chrome.rb

```diff
@@ -24,7 +24,28 @@ def initialize(info = {})
           'sinn3r',     # Metasploit post module
           'Kx499',      # x64 support
           'mubix'       # Parse extensions
-        ]
+        ],
+        'Compat' => {
+          'Meterpreter' => {
+            'Commands' => %w[
+              core_channel_close
```

Comment:
On L122, `rg` is set to railgun which is then used. I think that's a known limitation with the Rubocop implementation, in which case do we just want to accept that inaccuracy or add it in manually as a requirement.

Reply:
> rg is set to railgun which is then used

Thanks! PR to update the codebase to be more consistent in using railgun without an intermediate local variable being used: https://github.com/rapid7/metasploit-framework/pull/15713

 
> add it in manually as a requirement

Rubocop will be generating list entirely, if you add a command that isn't part of the list as identified by the automated rules, it will be removed. This ensures people don't just copy/paste metadata from another module, and not actually use the corresponding function calls.